README.md

```diff
@@ -1,14 +1,57 @@
 # SEF academix-frontend
 
+This build needs two modules
+1. Backend from sef-core
+    Contains the following APIs,
+    1. Multiverse Programme
+    
+2. Frontend from academix-frontend
+     Contains the academix home
+    
 ## Setup Development Environment
 
 ### Prerequisites
 1. Tomcat 9
 2. Maven
+3. MySQL 8
 
-### Building the project
-
-1. Build the project using maven
+### Building the backend 
```

Comment:
Add a new heading named Building the SEF-Core or something like that.

And mention why we need this